pkg/controller/controller_instance_test.go

```diff
@@ -269,6 +276,11 @@ func TestReconcileInstanceWithParameters(t *testing.T) {
 	// One single action comes from getting namespace uid
 	kubeActions := fakeKubeClient.Actions()
 	assertNumberOfActions(t, kubeActions, 1)
```

Comment:
Same deal here - action check for kube is in the middle of action check for catalog